src/vsr/replica.zig

```diff
@@ -3868,10 +3922,19 @@ pub fn ReplicaType(
             };
         }
 
-        /// Starting from the latest journal entry, backfill any missing or disconnected headers.
-        /// A header is disconnected if it breaks the chain with its newer neighbor to the right.
-        /// Since we work back from the latest entry, we should always be able to fix the chain.
-        /// Once headers are connected, backfill any dirty or faulty prepares.
+        /// Repair. Each step happens in sequence — step n+1 executes when step n is done.
+        ///
+        /// 1. Advance the head op to `op_repair_max = min(op_checkpoint_trigger, commit_max)`.
```

Comment:
Maybe rename `commit_max_limit` to `op_repair_max` in the body of fn?

Reply:
Or is it from the follow up PR?

Reply:
It is already called `op_repair_max` in the follow-up PR — it ends up being refactored into a (very short) function since it gets reused elsewhere too.